### PR TITLE
wire up the get started to the instructions

### PR DESCRIPTION
**Pull request by dealingwith**

### CODE REVIEW

File: app/assets/javascripts/application.js.coffee [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      params[key] = decodeURIComponent(val)
+
+    params
+
+  $('#get-started-shrugging').click ->
+    window.location = '/instructions?account=' + $('#github-account').val() + '&repo=' + $('#github-repository').val()
+
+  params = getParams()
+  $('#link-to-repo').attr 'href', 
+    'https://github.com/' + params.account + '/' + params.repo + '/settings/collaboration'
```
Comment (seanlinsley) [inline app/assets/javascripts/application.js.coffee]: Well I was thinking this link would be constructed from Rails, but ¯\_(ツ)_/¯

Comment (dealingwith) [inline app/assets/javascripts/application.js.coffee]: Oh, yeah. It should be :(

Comment (dealingwith) [inline app/assets/javascripts/application.js.coffee]: I will fix that up really quick.

Comment (dealingwith) [inline app/assets/javascripts/application.js.coffee]: I was in JS mode! :laughing:

**Pull request merged**